src/Features/CSharp/Portable/ConvertSwitchStatementToExpression/ConvertSwitchStatementToExpressionDiagnosticAnalyzer.AnalysisResult.cs

```diff
@@ -7,16 +7,16 @@ namespace Microsoft.CodeAnalysis.CSharp.ConvertSwitchStatementToExpression
 {
     internal sealed partial class ConvertSwitchStatementToExpressionDiagnosticAnalyzer
     {
-        private readonly struct AnalysisResult
+        private enum AnalysisResultKind
         {
-            private enum AnalysisResultKind
-            {
-                Neutral,
-                Failure,
-                Return,
-                Assignment,
-            }
+            Neutral,
+            Failure,
+            Return,
+            Assignment,
+        }
```

Comment:
no need for a 'throw' kind?

Reply:
Only if we want to offer the fix on a switch statement that throw in all cases. unlikely.